frontend/ui/data/css_tweaks.lua

```diff
@@ -1048,23 +956,73 @@ This tweak can be duplicated as a user style tweak when books contain footnotes
             }
             for __, rem in ipairs( { 1.0, 0.9, 0.85, 0.8, 0.75, 0.7, 0.65 } ) do
                 local pct = rem * 100
+                local fb2_notes_rem = 0.75 * rem / 0.8
+                local fb2_comments_rem = 0.85 * rem / 0.8
```

Comment:
If going with that, you'll probably want to format these to 2 digits after the dot.